ios/facebook_app_events.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'facebook_app_events'
-  s.version          = '0.0.1'
```

Comment:
Thank you for your feedback, but what you change is not according to what we need. Please check your PR first. because you just changed s.version, not the FBSDKCoreKit.